polldaddy-shortcode.php

```diff
@@ -292,7 +285,7 @@ function polldaddy_shortcode( $atts ) {
 					$inline = false;
 
 				$survey      = preg_replace( '/[^a-f0-9]/i', '', $survey );
-				$survey_url  = esc_url( "http://polldaddy.com/s/{$survey}" );
+				$survey_url  = esc_url( "https://survey.fm/{$survey}" );
```

Comment:
I don't think these domains will work, can I see an example of what this will look like, so we can test this out?

Reply:
Sure - these are for the old base36 ID's which can be used without the username and redirect to the 'correct' url format.

For example https://survey.fm/BB004BB8431FACB8 will redirect to https://ice9js.survey.fm/custom-css-survey.

Reply:
Perfect, good to know :)